lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -57,24 +57,28 @@
     width: 100%;
   }
   .exam-timer {
-    @include line-height(39);
     background-color: rgb(229, 234, 236);
-    padding-left: 42px;
-    padding-right: 32px;
+    padding: $baseline ($baseline*2);
     border-left: 4px solid $m-blue-l1;
     margin: 0 auto;
     color: $gray-d2;
     @include font-size(14);
+    .exam-text {
+      display: inline-block;
+      width: calc(100% - 250px);
+    }
     a {
       color: rgb(9, 121, 186);
     }
-    span.pull-right {
+    div.pull-right {
       color: $gray-d1;
-      @include line-height(39);
       b {
         color: $gray-d3;
       }
     }
+    div#turn_in_exam_id {
```

Comment:
Don't use IDs or HTML elements in selectors. Add a new class ".turn-in-exam" maybe?